app/assets/stylesheets/add.css

```diff
@@ -0,0 +1,15 @@
+h2{
```

Comment:
How did you decide on the file name 'add.css'? If you just had to pick something I understand, naming things is one of the [hardest things in programming](https://martinfowler.com/bliki/TwoHardThings.html) but I think using the name of the page i.e. index makes the most sense.